broker.go

```diff
@@ -84,50 +92,44 @@ func handleDefaultPort(w http.ResponseWriter, r *http.Request) {
 }
 
 func handleMessage(s *discordgo.Session, m *discordgo.MessageCreate) {
-	// Ignore all messages created by the bot itself
-	// This isn't required in this specific example but it's a good practice.
+	/* Validation */
 	if m.Author.ID == s.State.User.ID {
```

Comment:
We should still have a comment that indicates that this validation is to ignore self-messages, since the s.State.User.ID isn't very explicitly about "whose" User ID it is.

Reply:
Updated in latest commit.